earthmover/__main__.py

```diff
@@ -167,6 +172,9 @@ def main(argv=None):
     if not args.config_file:
         logger.error("config file not specified with `-c` flag, and no default {" + ", ".join(DEFAULT_CONFIG_FILES) + "} found")
 
+    if args.set and len(args.set)%2 != 0: # odd number of overrides
+        logger.error("overrides specified with --set must be followed by an even number of strings (key value key value ...)")
+
```

Comment:
I'd recommend converting the even list of arguments to a dictionary to pass into Earthmover.

```
overrides = dict(zip(args.set[::2], args.set[1::2]))
```

Reply:
That is better, thanks. I've revised this, marking resolved here.